src/gtk/widgets/recent_info.rs

```diff
@@ -141,23 +142,22 @@ impl RecentInfo {
 
     pub fn has_group(&self, group_name: &str) -> bool {
         unsafe {
-            to_bool(
-                ffi::gtk_recent_info_has_group(GTK_RECENT_INFO(self.unwrap_widget()),
-                                               group_name.borrow_to_glib().0))
+            to_bool(ffi::gtk_recent_info_has_group(GTK_RECENT_INFO(self.unwrap_widget()),
+                group_name.borrow_to_glib().0))
         }
```

Comment:
The biggest problem for me here is touching stuff that wasn't broken, doesn't benefit from changing and on top of that becomes harder to read at least to me.